python/cuml/common/import_utils.py

```diff
@@ -115,3 +116,20 @@ def dummy_function_always_false(*args, **kwargs):
 
 class DummyClass(object):
     pass
+
+
+def check_cupy8(conf=None):
+    def check_cupy8_dec(func):
+        @wraps(func)
+        def inner(*args, **kwargs):
+            if check_min_cupy_version('8.0'):
```

Comment:
any way we could make this work on release candidates too to test in the meantime?

Reply:
You can absolutely test it by installing a beta version, if that's your question. `pip install cupy==8.0.0b4`

Reply:
So 8.0.0b4 will pass this test and allow the function to be used? Sounds good by me.